src/components/indicator/Indicator.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+import Indicator from './Indicator';
+
+import notes from './Indicator.stories.md';
+
+export const basic = () => (
+    <Indicator>
+        <span>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+            dolore magna aliqua.
```

Comment:
This text is cut off in the screenshot. Is there a way to view all the text contained within the indicator?

Reply:
The UX guideline is to have a tooltip on hover, which can be done via `title` attribute. But given that the `children` accepts a react node and not a string, I wasn't sure whether it should expose extra prop or let the consumer of the component wrap with a proper `title` text like below.
```
<div title="translated text">
  <Indicator>
    <FormattedMessage defaultMessage="translated text" />
  </Indicator>
</div>
```
Do you think it's worth exposing a string type prop for this case?

Reply:
I think we'd want to use the Tooltip component rather than the browser's built-in one, which is slow to appear and hard to read